wrappers/python/tests/nntile_core/test_tensor_strassen.py

```diff
@@ -0,0 +1,113 @@
+# @copyright (c) 2022-2023 Skolkovo Institute of Science and Technology
+#                           (Skoltech). All rights reserved.
+#
+# NNTile is software framework for fast training of big neural networks on
+# distributed-memory heterogeneous systems based on StarPU runtime system.
+#
+# @file wrappers/python/tests/nntile_core/test_tensor_strassen.py
+# Test for tensor::strassen<T> Python wrapper
+#
+# @version 1.0.0
+# @author Aleksandr Mikhalev
+# @date 2023-12-18
+
+# All necesary imports
+import nntile
+import numpy as np
+import itertools
+
+# Set up StarPU configuration and init it
+config = nntile.starpu.Config(1, 0, 0)
+# Init all NNTile-StarPU codelets
+nntile.starpu.init()
+# Define list of tested types
+dtypes = [np.float32, np.float64]
+# Define mapping between numpy and nntile types
+Tensor = {np.float32: nntile.tensor.Tensor_fp32, np.float64: nntile.tensor.Tensor_fp64}
+# Define mapping between tested function and numpy type
+strassen = {
+    np.float32: nntile.nntile_core.tensor.strassen_fp32,
+    np.float64: nntile.nntile_core.tensor.strassen_fp64,
+}
+
+
+# Helper function returns bool value true if test passes
+def helper(dtype, tA, tB, matrix_shape, shared_size, tile_size, alpha, beta, batch=3):
```

Comment:
Instead of describing function with a comment, write a oneliner in a docstring as follows.

def helper(dtype, tA, tB, matrix_shape, shared_size, tile_size, alpha, beta, batch=3):
    ""Helper function returns bool value true if test passes."""